modules/sandboxed-containers-deleting-operator.adoc

```diff
@@ -36,3 +37,24 @@ CSV_NAME=$(oc get csv -n openshift-sandboxed-containers-operator -o=custom-colum
 ----
 $ oc delete csv ${CSV_NAME} -n openshift-sandboxed-containers-operator
 ----
+
+. Fetch the {sandboxed-containers-first} Operator group name by running the following command:
++
+[source, terminal]
+----
+$ OG_NAME=$(oc get operatorgroup -n openshift-sandboxed-containers-operator -o=jsonpath={..name})
+----
+
+. Delete the {sandboxed-containers-first} Operator group name by running the following command:
++
+[source, terminal]
+----
+$ oc delete operatorgroup ${OG_NAME} -n openshift-sandboxed-containers-operator
+----
+
+. Delete the {sandboxed-containers-first} namespace by running the following command:
```

Comment:
Why do we have that step here and also a separate section for deleting the namespace?

Reply:
@c3d  Thanks for your feedback Christophe. It's a mistake - the command is included in the Deleting an Operator section. (I initially created the section and decided not to use it. I'll delete it.)

Reply:
Fixed. I deleted the separate section.